src/lib/forms/components/ExceptionsField.js

```diff
@@ -0,0 +1,78 @@
+import { AccordionField } from '@forms/core/AccordionField';
+import { ArrayField } from '@forms/core/ArrayField';
+import { GroupField } from '@forms/core/GroupField';
+import { StringField } from '@forms/core/StringField';
+import React, { Component } from 'react';
+import { DeleteActionButton } from './DeleteActionButton';
+import { BooleanField } from '@forms/core/BooleanField';
+import { DateInputField } from '@forms/core/DateTimeFields/DateInputField';
+import { Divider, Grid } from 'semantic-ui-react';
+
+export class ExceptionsField extends Component {
```

Comment:
Minor: naming a bit too generic `src/lib/forms/components/ExceptionsField.js`, is it only for the closures?
It might not belong to the re-usable generic components for forms, but maybe only to the "closure" module?